integrations/purchase/bridge/bridge-api-categories.js

```diff
@@ -94,12 +92,12 @@ const idToCategory = {
   248: PURCHASE_CATEGORY_STORE_PERSONAL_CARE, // Cosmetics
   235: PURCHASE_CATEGORY_STORE_PERSONAL_CARE, // Hairdresser
 
-  161: PURCHASE_CATEGORY_HOUSING, // - Home
+  161: null, // - Home (not sure about the subcategory in PURCHASE_CATEGORY_HOUSING)
```

Comment:
I have verified that this was not affected by my Jest caching issue. We do not have a carbon footprint for `PURCHASE_CATEGORY_HOUSING` (`HOUSING, WATER, ELECTRICITY, GAS AND OTHER FUELS`).